### DISCUSSION TITLE
Connecting an electon js app to meshcentral

### DISCUSSION THREAD

**Discussion by rajatheynetech** [Ylianst/MeshCentral#4331] [Q&A] [answered]
Hi, I'm quite new to mesh central. I have a desktop app which is developed using electron js and node js. On the same system, I have installed the MESH AGENT.  I'm planning to channel the messages and alerts from the mesh central server through this electron app rather than the  MESH AGENT (Please find the attached image). In my case, the MESH AGENT will only act as a service which will allow the MESH SERVER to communicate with my system.  Any other possible way?. Please help.
<img width="475" alt="Screenshot 2022-07-27 at 2 08 38 PM" src="https://user-images.githubusercontent.com/67583922/181203952-090e108f-dd7f-435d-b298-f1a32560f2fe.png">

**Comment by jsastriawan** [comment, marked as answer]
Have a look at a custom router app I wrote. Yes it is possible.

https://github.com/jsastriawan/mc2router-electron

**Comment by rajatheynetech** [comment]
@jsastriawan Thank you very much.

**Comment by Ylianst** [comment]
Nice. You may also want to consult #4292 for information on how to build applications (including desktop applications) that connect with MeshCentral. I offer a bunch of into in that thread.
